api-test/src/main/resources/idRepository/IdVidMetadataSearch/IdVidMetadataSearch.yml

```
@@ -0,0 +1,73 @@
+    inputTemplate: idRepository/IdVidMetadataSearch/IdVidMetadataSearch
+    outputTemplate: idRepository/IdVidMetadataSearch/IdVidMetadataSearchResult
+    input: '{
+      "id": "mosip.idrepo.idvid.metadata",
+      "version": "1.0",
+      "requestTime": "$TIMESTAMP$",
+      "individualId": "$ID:AddIdentity_withValidParameters_smoke_Pos_UIN$",
+	  "idType": "$REMOVE$"
+}'
+    output: '{
+}'
+  IdRepository_IdVidMetadataSearch_With_Required_Details:
```

YAMLlint (1.38.0)

[error] duplication of key "IdRepository_IdVidMetadataSearch_With_Required_Details" in mapping

(key-duplicates)